dev-lang/python/python-3.13.0_rc1_p3.ebuild

```diff
@@ -125,6 +126,16 @@ pkg_pretend() {
 		ewarn "you can reproduce the problem with dev-lang/python[gil,-jit].  Instead,"
 		ewarn "please consider reporting freethreading / JIT problems upstream."
 	fi
+
+	if use kernel_linux && [[ "${MERGE_TYPE}" != "binary" ]]; then
+		if use pgo || use test; then
```

Comment:
I think we don't want to enforce config options for `USE=pgo` — we should rather skip that specific test.